content/3_Silver/Prefix_Sums_2.mdx

```diff
@@ -64,6 +69,58 @@ int main(){
 
 </CPPSection>
 
+<JavaSection>
+
+### Java Implementation
+
+```java
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+
+public class MaxSubSum {
+	public static void main(String[] args) throws IOException {
+		BufferedReader read = new BufferedReader(new InputStreamReader(System.in));
+		read.readLine();
+		int[] arr = Arrays.stream(read.readLine().split(" ")).mapToInt(Integer::parseInt).toArray();
```

Comment:
lines 84 and 86 are too long

Reply:
How should I split this up?

Reply:
uhh i'm not sure... i guess you could probably leave it as it is, since it already wraps in the code block and splitting it up won't solve that issue